### PR TITLE
Ensure milo can find builds triggered by cocoon backend.

### PR DESCRIPTION
**Pull request by godofredoc**
This change makes the builds created by cocoon backend discoverable by
the milo queries.

### CODE REVIEW

File: app_dart/lib/src/request_handlers/query_github_graphql.dart [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+class QueryGithubGraphql extends ApiRequestHandler<Body> {
+  const QueryGithubGraphql(
+    Config config,
+    AuthenticationProvider authenticationProvider,
+  ) : super(config: config, authenticationProvider: authenticationProvider);
+
+  @override
+  Future<Body> post() async {
+    final String requestDataString = String.fromCharCodes(requestBody);
+    log.error(requestDataString);
```
Comment (yusufm) [inline app_dart/lib/src/request_handlers/query_github_graphql.dart:30]: should this be a lower log level (info, debug?)

**Review by CaseyHillers — COMMENTED** [review]
How does this PR surface builds in MILO?

File: app_dart/lib/src/request_handlers/query_github_graphql.dart (hunk 1 above)
Comment (godofredoc) [reply app_dart/lib/src/request_handlers/query_github_graphql.dart]: uppps wrong file

> How does this PR surface builds in MILO?



> How does this PR surface builds in MILO?

wrong file, uploaded the correct one.

**Review by CaseyHillers — APPROVED** [review]
LGTM - Thanks for the fix!

**Pull request merged**